Tests/SwiftProtobufTests/Test_TextFormat_Performance.swift

```diff
@@ -0,0 +1,100 @@
+// Tests/SwiftProtobufTests/Test_TextFormat_proto3.swift - Exercise proto3 text format coding
+//
+// Copyright (c) 2022 - 2022 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/main/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// This is a set of tests for text format protobuf files.
+///
+// -----------------------------------------------------------------------------
+
+import XCTest
+import SwiftProtobuf
+
+class Test_TextFormat_Performance: XCTestCase, PBTestHelpers {
+    typealias MessageTestType = Fuzz_Testing_Message
+
+    // Each of the following should be under 1s on a reasonably
+    // fast machine (originally developed on an M1 MacBook Pro).
+    // If they are significantly slower than that, then something
+    // may be amiss.
+
+    func testEncoding_manyMapsEncoding_shouldBeUnder1s() {
+        let repeats = 50000
+
+        let child = (
+            "repeated_message {\n"
```

Comment:
You probably can use a multi line string with the current min requirements.

Reply:
Personally, I'm often confused about how indentation works for multi line strings, so I prefer to be a bit more explicit.